dpytools/http/README.md

```diff
@@ -68,4 +68,38 @@ response = http_client.post(
 )
 ```
 
-If the `POST` request fails for a network-related reason, this will raise an `HTTPError`.
\ No newline at end of file
+If the `POST` request fails for a network-related reason, this will raise an `HTTPError`.
+
+### UploadClient
+
+The `UploadClient` class facilitates the process of uploading a file to an AWS S3 bucket by splitting the file into chunks and transmitting these chunks individually. This is done via the `upload()` method.
+
+A new `UploadClient` object can be created by passing an `upload_url`:
+
+```python
+from dpytools.http.upload import UploadClient
+
+upload_client = UploadClient(upload_url="http://example.org/upload")
+```
+
+To access the DP Upload Service, a Florence access control token must be provided. For security purposes this should be set in the command line using an environment variable:
```

Comment:
just say its an environment variable please, command. line is one of many ways to set them.